frame/bounties/src/lib.rs

```diff
@@ -475,10 +544,26 @@ decl_module! {
 					BountyStatus::CuratorProposed { ref curator } => {
 						ensure!(signer == *curator, Error::<T>::RequireCurator);
 
+						// Reserve the Curator deposit
 						let deposit = T::BountyCuratorDeposit::get() * bounty.fee;
 						T::Currency::reserve(curator, deposit)?;
 						bounty.curator_deposit = deposit;
 
+						// Update the Curator info to Subbounties list
+						if bounty.activesubbounty.len() > 0 {
```

Comment:
update curator of parent bounty shouldn't modify curator of sub bounties

Reply:
Subcurator of subbounty is not touched, Just updating the curator state of parent bounty. As per state machine, there could be possibility like when bounty is added with subbounty and active with subcurator, council may change the curator of parent bounty.

Reply:
why are you storing curator again in sub bounties?

Reply:
Good question Bryan,

Following subbounty api has dependency on curator state info of parent bounty.

propose_subcurator() | accept_subcurator() 
| unassign_subcurator() | claim_subbounty()

This state info is used for ...

1, To ensure origin of the subbounty call is from bounty curator.
2, To distinguish between curator & subcurator. for eg use-case like
   if curator self assign himself as subcurator for subbounty, curator fee 
   needs to be considered as zero, similarly few more use-cases are there.

Initially thought of extracting this info from "Struct Bounty" 
using bounty_id. but to take advantage of performance over storage size.
added extra "curator" field to "Struct SubBounty" such that i can 
avoid loading "Struct Bounty".

Could be I am wrong here, for simple state info verification, 
if loading of "Struct Bounty" is simpler, then I move for it.

Reply:
I will prefer load bounty again over duplicate the data.

Reply:
Thanks Bryan, then I make changes to suggested one.

Reply:
Suggested changes are Done :+1: